components/chatbot/controller.js

```diff
@@ -123,30 +95,21 @@ function Controller() {
 
       if (!response.body)
         throw Error("ReadableStream not yet supported in this browser.");
-      console.log("This is debug input text", inputText);
       const reader = response.body.getReader();
 
       let accumulatedResponse = "";
 
       reader.read().then(async function process({ done, value }) {
         if (done) {
-          let sourceStatus = await getSourceStatus();
-          console.log("This is source status", sourceStatus);
-          let fileData;
-          if (sourceStatus == "Document_Display") {
-            fileData = await displayRelevantFile(
-              chatId,
-              inputText,
-              accumulatedResponse
-            );
-          } else {
-            fileData = await getRelevantFile(
```

Comment:
function displayRelevantFile and getRelevantFile are basically the same function. So here changed to a single function getRelevantFile with a different parameter sourceStatus.